disperser/metrics.go

```diff
@@ -78,6 +79,15 @@ func NewMetrics(reg *prometheus.Registry, httpPort string, logger logging.Logger
 			},
 			[]string{"method"},
 		),
+		BlobLatency: promauto.With(reg).NewGaugeVec(
```

Comment:
Using gauge instead of summary, to reduce the number of time series -- it should be sufficient to just have gauge to get insights for this latency breakdown